proto/xudrpc.proto

```diff
@@ -379,10 +383,12 @@ message PlaceOrderRequest {
   OrderSide side = 5 [json_name = "side"];
 }
 message PlaceOrderResponse {
-  // A list of orders matching the newly placed order
-  repeated OrderMatch matches = 1 [json_name = "matches"];
+  // A list of own orders (or portions thereof) that matched the newly placed order
+  repeated Order internal_matches = 1 [json_name = "internal_matches"];
+  // A list of swap results of peer orders (or portions thereof) that matched newly placed order
```

Comment:
I feel like the "(or portions thereof)" isn't necessary here, to me here it's more clear that this is just the result of a swap and not a particular order where we might be concerned with the full or remaining quantity of the order.